src/styles/searchResults.css

```diff
@@ -0,0 +1,19 @@
+.search-results {
+    display: flex;
+    flex-direction: row;
```

Comment:
As before this property is set as a default so you can remove this, also the `align-items` property as that is for vertically oriented items, which you arent using here